lib/catalog/martyrology.ts

```diff
@@ -1818,6 +1878,13 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       canonizationLevel: CanonizationLevels.Saint,
       name: 'Jacinta Marto',
     },
+    jacques_jules_bonnaud_martyr: {
+      canonizationLevel: CanonizationLevels.Blessed,
+      name: 'Jacques Jules Bonnaud',
+      titles: [Titles.Martyr],
```

Comment:
According to [Wikipedia](https://fr.wikipedia.org/wiki/Jacques_Bonnaud), he was a priest, therefore the key should be renamed to `jacques_jules_bonnaud_priest` and `Titles.Priest` title added?

```suggestion
    jacques_jules_bonnaud_priest {
      canonizationLevel: CanonizationLevels.Blessed,
      name: 'Jacques Jules Bonnaud',
      titles: [Titles.Priest, Titles.Martyr],
```